src/module/chat/chatrollhelpers.ts

```diff
@@ -195,14 +171,13 @@ function calculateSFMoveResultText(
   move: IronswornItem
 ) {
   const data = move.data as SFMoveDataProperties
-  const outcomeKey = {
-    [ROLL_OUTCOME.MISS]: 'Miss',
-    [ROLL_OUTCOME.WEAK_HIT]: 'Weak Hit',
-    [ROLL_OUTCOME.STRONG_HIT]: 'Strong Hit',
-  }[type]
-  let outcome = data.data.Outcomes?.[outcomeKey]
-  if (match) outcome = outcome?.['With a Match'] ?? outcome
-  return outcome?.Text
+  const dfOutcomeKey = DfRollOutcome[type]
+  const parentOutcome = data.data.Outcomes?.[dfOutcomeKey]
+
+  if (match) {
+    return parentOutcome['With a Match']?.Text ?? parentOutcome.Text
+  }
+  return parentOutcome.Text
```

Comment:
Why the name change? What is this a parent or child of?

Reply:
"parent" in the semantic sense ("Strong hit" is parent to "Strong hit with a match"), but i reckon you're right on it being confusing -- i'll just rename it to `outcome`.